src/models/PartialFormSubmission.php

```diff
@@ -25,42 +32,64 @@ class PartialFormSubmission extends SubmittedForm
     private static $table_name = 'PartialFormSubmission';
 
     private static $db = [
-        'IsSend' => 'Boolean(false)'
+        'IsSend' => 'Boolean(false)',
```

Comment:
Could you align the `=>`? (Sorry, I'm really picky :D )

Reply:
Yep sure, it did come across my mind if I needed to